electron/browser/sessions.ts

```
@@ -0,0 +1,76 @@
+
+      dialog
+        .showMessageBox({
+          message: `"${minifiedUrl}" wants to open "${externalAppName}".`,
+          buttons: ["Cancel", "Open", "Always Open"]
+        })
+        .then((response) => {
+          switch (response.response) {
+            case 2:
+              if (externalURL) {
+                setAlwaysOpenExternal(requestingURL, externalURL);
+              }
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)